<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(38, 41, 45);
border-radius: 10px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_header">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color:rgb(28, 31, 35)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>6</y>
          <width>148</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(160, 160, 160)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Чисельні методи</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_exp">
        <property name="geometry">
         <rect>
          <x>373</x>
          <y>8</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	border-radius: 7px;
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(0, 111, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 153, 0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_close">
        <property name="geometry">
         <rect>
          <x>394</x>
          <y>8</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border-radius: 8px;
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(175, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(229, 0, 0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_credits">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>630</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(208, 209, 185);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>&lt;strong&gt;Created:&lt;/strong&gt; Семчук Олег</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>421</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>  Лабораторні роботи</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>211</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style:  solid;
	border-radius: 15px;
	border-width: 5px;
	color: rgba(255, 255, 255, 200);
	border-color:rgb(53, 61, 70);
	background-color: rgb(67, 72, 79);
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(99, 107, 117);
}

QPushButton:pressed{
	background-color: rgb(47, 51, 56);
}
</string>
          </property>
          <property name="text">
           <string>Лаба 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style:  solid;
	border-radius: 15px;
	border-width: 5px;
	color: rgba(255, 255, 255, 200);
	border-color:rgb(53, 61, 70);
	background-color: rgb(67, 72, 79);
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(99, 107, 117);
}

QPushButton:pressed{
	background-color: rgb(47, 51, 56);
}
</string>
          </property>
          <property name="text">
           <string>Лаба 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style:  solid;
	border-radius: 15px;
	border-width: 5px;
	color: rgba(255, 255, 255, 200);
	border-color:rgb(53, 61, 70);
	background-color: rgb(67, 72, 79);
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(99, 107, 117);
}

QPushButton:pressed{
	background-color: rgb(47, 51, 56);
}
</string>
          </property>
          <property name="text">
           <string>Лаба 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style:  solid;
	border-radius: 15px;
	border-width: 5px;
	color: rgba(255, 255, 255, 200);
	border-color:rgb(53, 61, 70);
	background-color: rgb(67, 72, 79);
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(99, 107, 117);
}

QPushButton:pressed{
	background-color: rgb(47, 51, 56);
}
</string>
          </property>
          <property name="text">
           <string>Лаба 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style:  solid;
	border-radius: 15px;
	border-width: 5px;
	color: rgba(255, 255, 255, 200);
	border-color:rgb(53, 61, 70);
	background-color: rgb(67, 72, 79);
	text-align: center;
}

QPushButton:hover{
	background-color: rgb(99, 107, 117);
}

QPushButton:pressed{
	background-color: rgb(47, 51, 56);
}
</string>
          </property>
          <property name="text">
           <string>Лаба 6</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
